src/js/components/Pagination/PaginationSummary.js

```diff
@@ -0,0 +1,34 @@
+import React, { useContext } from 'react';
+import { Box } from '../Box';
+import { Text } from '../Text';
+import { MessageContext } from '../../contexts/MessageContext';
+
+export const PaginationSummary = ({
+  messages,
+  numberItems,
+  page,
+  step,
+  ...rest
+}) => {
+  const { format: formatMessage } = useContext(MessageContext);
+  return (
+    <Box {...rest}>
```

Comment:
I see the `...rest` spread on this Box. Just confirming that's intentional? I know there had been discussion about on Box vs Text.

Reply:
That was my suggestion

Reply:
yes we wanted the `...rest` on the Box then if we want to enhance Text we could do something like `textProp` in theme that accept any textProps or something similar to `DropProps` I figured we can enhance that and doesn't necessarily have to be in the first pass

Reply:
Sounds good. I'm fine with waiting until use case calls for it. Don't need it for first pass.

Reply:
sounds great!